src/ProductStore.php

```diff
@@ -75,6 +86,77 @@ public function set($key, $value, $renewal = NULL) {
       ->execute();
   }
 
+  /**
+   * Sets overrides for the given key.
+   *
+   * @param string $key
+   *   Key.
+   * @param array $overrides
+   *   The overrides.
+   *
+   * @throws \Exception
+   */
+  public function setOverride($key, array $overrides) {
+    $value = $this->get($key);
+    $this->connection->merge($this->table)
+      ->keys([
+        'name' => $key,
+        'collection' => $this->collection
+      ])
+      ->fields([
+        'overrides' => $this->serializer->encode($overrides),
+        'value' => $this->serializer->encode($value),
```

Comment:
value should be nullable instead, so we dont have to do $value = $this->get($key);